lib/ts/recipe/emailpassword/emaildelivery/services/smtp/passwordReset.ts

```diff
@@ -0,0 +1,380 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+import { TypeEmailPasswordPasswordResetEmailDeliveryInput } from "../../../types";
+import { GetContentResult } from "../../../../../ingredients/emaildelivery/services/smtp";
+import Supertokens from "../../../../../supertokens";
+export default function getPasswordResetEmailContent(
+    input: TypeEmailPasswordPasswordResetEmailDeliveryInput
+): GetContentResult {
+    let supertokens = Supertokens.getInstanceOrThrowError();
+    let appName = supertokens.appInfo.appName;
```

Comment:
Can you pass appInto to the input as opposed to getting the supertokens instance? this will cause issues when we have multi tenancy...